rust/remotivelabs-broker/README.md

```diff
@@ -0,0 +1,21 @@
+# RemotiveLabs Broker
+
+Rust library with gRPC-bindings and utility functions.
+
+gRPC bindings are generated from [this source](../../proto/) when compiling the project.
+
+This project is contributed by _Niclas Lind_ (@niclaslind).
+
+## Publish / Subscribe example
+See `examples/pub_client.rs` and `examples/sub_client.rs` for a sample on how to use the library.
```

Comment:
mention that this example is using the virtual namespace

Reply:
Added mention